Makefile

```diff
@@ -36,6 +36,9 @@ huawei-cloud-controller-manager: $(SOURCES)
 clean:
 	rm -rf huawei-cloud-controller-manager
 
+lint:
+	go run github.com/golangci/golangci-lint/cmd/golangci-lint@v1.49.0 run ./...
+
```

Comment:
The `lint` is only part of `verify`. We can add a new script, like https://github.com/karmada-io/karmada/blob/master/hack/verify-staticcheck.sh,  and merge it into `hack/verify.sh`.